frontend-maven-plugin/src/main/java/com/github/eirslett/maven/plugins/frontend/mojo/AbstractFrontendMojo.java

```diff
@@ -74,7 +75,7 @@ private boolean isTestingPhase() {
         return "test".equals(phase) || "integration-test".equals(phase);
     }
 
-    protected abstract void execute(FrontendPluginFactory factory) throws FrontendException;
+    protected abstract void execute(FrontendPluginFactory factory) throws Exception;
```

Comment:
Do we need to generalise the exception types here? (ie. in the context of [m1](https://stackify.com/common-mistakes-handling-java-exception/))